packages/venia-ui/lib/components/MiniCart/miniCart.css

```diff
@@ -49,7 +49,7 @@
     overflow: hidden;
     padding: 1rem;
     position: fixed;
-    right: 0.5rem;
+    right: -100vw;
```

Comment:
Hmm, this doesn't feel like the solution. I think the problem here is that both `.root` and `.contents` have `position: fixed`, which is relative to the viewport. The latter should probably have `position: absolute`, so that it's relative to the former. That way, when the parent moves offscreen, it'll take the child with it.

```stylus
.root
  position: fixed

.contents
  position: absolute
```

Reply:
Awesome, thanks. Resolved in https://github.com/magento/pwa-studio/pull/2547/commits/c9812fd6bd0fad45edf71cd89bb55b84af3bfb62.